agent/cmd/edgemesh-agent/app/server.go

```diff
@@ -164,8 +164,8 @@ func registerModules(c *config.EdgeMeshAgentConfig, ifm *informers.Manager) []er
 func prepareRun(c *config.EdgeMeshAgentConfig, ifm *informers.Manager) error {
 	// set dns and proxy modules listenInterface
 	if c.Modules.EdgeDNSConfig.Enable || c.Modules.EdgeProxyConfig.Enable {
-		if err := commonutil.CreateDummyDevice(c.CommonConfig.DummyDeviceName, c.CommonConfig.DummyDeviceIP); err != nil {
-			return fmt.Errorf("create dummy device %s err: %v", c.CommonConfig.DummyDeviceName, err)
+		if err := commonutil.CreateEdgeMeshDevice(c.CommonConfig.DummyDeviceName, c.CommonConfig.DummyDeviceIP); err != nil {
+			return fmt.Errorf("create edgemesh device %s err: %v", c.CommonConfig.DummyDeviceName, err)
```

Comment:
It is better to use %w for wrapping error, and the error message we cloud use 
```suggestion
			return fmt.Errorf("failed to create edgemesh device %s: %w", c.CommonConfig.DummyDeviceName, err)
```

Reply:
fix